File: starting_filesets/C#/Leap Year/LeapYearTest.cs

using NUnit.Framework;

[TestFixture]
public class LeapYearTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new LeapYear().Answer;
        Assert.AreEqual(expected, actual);
    }
}

File: starting_filesets/PHP/Leap Year/LeapYearTest.php
<?php

require_once 'PHPUnit/Framework.php';
require_once 'LeapYear.php';

class LeapYearTest extends PHPUnit_Framework_TestCase
{
    public function testAnswer()
    {
        $this->assertEquals(6 * 9, answer());
    }
}

?>
